tests/unit/records/test_schemas_marcxml.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2016 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Zenodo Marcxml mapping test."""
+
+from __future__ import absolute_import, print_function
+
+import json
+
+from invenio_records_rest.serializers.json import JSONSerializer
+from zenodo.modules.records.serializers.schemas.marcxml import RecordSchemaMARC
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_records import Record
+
+
+def test_full_record(app, full_record):
+    """Test full record."""
+    full_record['alternate_identifiers'] = [
+        {
+            "identifier": "urn:lsid:ubio.org:namebank:11815",
+            "scheme": "lsid",
+            "relation": "alternateIdentifier",
+        },
+        {
+            "identifier": "2011ApJS..192...18K",
+            "scheme": "ads",
+            "relation": "alternateIdentifier",
+        },
+        {
+            "identifier": "10.1234/alternate.doi",
+            "scheme": "doi",
+            "relation": "alternateIdentifier",
+        },
+    ]
+    full_record['meetings'] = {
```

Comment:
I think this should be a list with a dict inside.

Please add a test to see that it validates. E.g.:
``assert Record(full_record).validate()``